### PR TITLE
Update controllerClient information

### PR DESCRIPTION
**Pull request by feerrenrut**
### Link to issue number:
None, discussion on mailing list:
https://groups.io/g/nvda-devel/message/46179

### Summary of the issue:
NVDA controller client information is outdated, and could be clarified.

### Description of how this pull request fixes the issue:
Clarify information, provide updated links.

### Testing strategy:
None

### Known issues with pull request:
None

### Change log entries:
None

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English

### CODE REVIEW

File: extras/controllerClient/readme.md [hunk 1]
```diff
@@ -8,30 +8,43 @@ The client API is implemented as a dll (dynamic link library). The functions in
...
 
-Note: The header and library files are built along with nvdaHelper. If you want to build them yourself, please see readme.txt in the NVDA source distribution for instructions. Alternatively, you can find here a [pre-built package](http://www.nvda-project.org/nvdaControllerClient/nvdaControllerClient_20100219.7z).
+**Note:**
 
-Versions of the libraries and headers are supplied for both x86 and x64 applications. They have been split in to x86 and x64 directories, respectivly.
+The header and library files are built along with nvdaHelper.
+If you want to build them yourself, please see `readme.txt` in the NVDA source distribution for instructions.
 
-Each directory contains the following files, where * denotes 32 for x86 and 64 for x64:
+Versions of the libraries and headers are supplied for both `x86` and `x64` applications. They have been split in to `x86` and `x64` directories, respectivly.
```
Comment (lukaszgo1) [inline extras/controllerClient/readme.md:16]: respectivly -> respectively
```suggestion
Versions of the libraries and headers are supplied for both `x86` and `x64` applications. They have been split in to `x86` and `x64` directories, respectively.
```

Comment (lukaszgo1) [inline extras/controllerClient/readme.md:16]: For some time now we also compile Controller Client for ARM64. Shouldn't it be mentioned as a supported architecture here?

File: extras/controllerClient/readme.md [hunk 2]
```diff
@@ -6,32 +6,47 @@ This client API allows an application to communicate with NVDA, in order to do s
...
 
 ## License
 
-The NVDA Controller Client API is licensed under the GNU Lesser General Public License (LGPL), version 2.1\. In simple terms, this means you can use this library in any application, but if you modify the library in any way, you must contribute the changes back to the community under the same license.
+The NVDA Controller Client API is licensed under the GNU Lesser General Public License (LGPL), version 2.1.
+In simple terms, this means you can use this library in any application, but if you modify the library in any way, you must contribute the changes back to the community under the same license.
 
-Please see license.txt in this directory for more details.
\ No newline at end of file
+Please see `license.txt` in this directory for more details.
```
Comment (seanbudd) [inline extras/controllerClient/readme.md:52]:
```suggestion
Please refer to [license.txt](./license.txt) for more details.
```

**Review by seanbudd — APPROVED** [review]
LGTM, one minor suggestion

**Comment by seanbudd** [comment]
apologies for the late review

**Pull request merged**